index.rst

```diff
@@ -1,57 +1,343 @@
+:tocdepth: 1
+
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+
+.. sectnum::
+
+.. note::
+
+   This tech note details some basic concepts about how users will interact with the Vera Rubin Observatory control system during commissioning and operations.
+
+
+.. _Introduction:
+
+Introduction
+============
+
+The construction, integration, commissioning and operation of observatories requires a significant amount of knowledge spanning multiple disciplines.
+However, all operational aspects are performed using some piece of the control software.
+The range of use-cases span from an engineer performing embedded low-level servo control to enable precise positioning of the mount, to an observer using a simple graphical display to evaluate the current weather conditions.
+
+To account for the large number of use-cases from all levels of users, the control system must simultaneously provide a high degree of flexibility and a user-friendly framework.
+The top-level functionality specifications that define much of the development of the observatory control system are:
+
+  - Regular operations routines should be easily accessible;
+  - Must include a highly customizable scripting capability must be available and able to run user-written code;
+  - An interactive mode, equivalent to a user sandbox space, must be available to allow development of routines on the fly as well as debugging;
+  - Capability to incorporate user-developed features;
+
+In the standard operational model used to perform the survey, the observatory will be largely driven by the observatory Scheduler.
+The Scheduler designates a target and a specially formatted script containing a series of operations to be performed (such as slewing the telescope and taking an image).
+The execution of the script handled by the control system while the user is able to observe (or interrupt) the progress via a graphical interface.
+Operators will be much more involved during the start-up and shutdown phases, as well as during afternoon calibrations.
+In the majority of cases, these routine operations are expected to be executed by the operator from a high-level interface of the observatory control software (as discussed in following sections).
+
+During integration, commissioning, and engineering time, there will be situations where non-standard, customized operations will need to be executed.
+These operations will often be exercised by specialists to a specific subsystem (e.g. M1M3) and require a knowledge base that differs from that of an observing specialist.
```

Comment:
I really liked this phrase. One suggestion, would it make sense to replace "specialists to a specific subsystem"  by "subsystem specialist"? It will match really well with the later use of "observing specialist", e.g.:

> These operations will often be exercised by subsystem specialists (e.g. M1M3) and require a knowledge base that differs from that of an observing specialist.